api/HTMLDialogElement.json

```diff
@@ -80,26 +89,10 @@
               "version_added": "79"
             },
             "firefox": {
-              "version_added": "78",
-              "flags": [
-                {
-                  "type": "preference",
-                  "name": "dom.dialog_element.enabled",
-                  "value_to_set": "true"
-                }
-              ],
-              "notes": "See <a href='https://bugzil.la/840640'>bug 840640</a>."
+              "version_added": "98"
```

Comment:
I think—to make sense with the parent feature—this should be two support statements in an array:

1. Added in 98
2. Added in 78, behind a flag

Reply:
done